packages/xo-web/src/common/xo/index.js

```diff
@@ -777,6 +777,12 @@ export const emergencyShutdownHosts = hosts => {
   }).then(() => map(hosts, host => emergencyShutdownHost(host)), noop)
 }
 
+export const assertConsistentHostServerTime = host =>
+  _call('host.assertConsistentHostServerTime', { host: resolveId(host) }).then(
```

Comment:
It's a public function which can be used on other cases. Simply return the promise to not change the behavior.